test/clj/rems/test_pdf.clj

```diff
@@ -162,7 +162,9 @@
                  [[:paragraph pdf/field-style "Text field with max length"]
                   [:paragraph "pdf test"]]
                  [[:paragraph pdf/field-style "Text area with max length"]
-                  [:paragraph "pdf test"]]]]]
+                  [:paragraph "pdf test"]]
+                 [[:paragraph {:spacing-before 8, :style :bold} "Phone number"]
```

Comment:
You should use `pdf/field-style` here as for the other cases. The idea is to avoid depending on the specific pdf styles and to only test the structure of the document.

Reply:
addressed